src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -2428,4 +2435,42 @@ static OrganizationUserStatusType GetPriorActiveOrganizationUserStatusType(Organ
 
         return status;
     }
+
+    public async Task CreatePendingOrganization(Organization organization, string ownerEmail)
+    {
+        organization.Enabled = false;
+        organization.Status = OrganizationStatusType.Pending;
+        await _organizationRepository.CreateAsync(organization);
+
+        var ownerOrganizationUser = new OrganizationUser
+        {
+            OrganizationId = organization.Id,
+            UserId = null,
+            Email = ownerEmail,
+            Key = null,
+            Type = OrganizationUserType.Owner,
+            Status = OrganizationUserStatusType.Invited,
+            AccessAll = true,
+            CreationDate = organization.CreationDate,
+            RevisionDate = organization.CreationDate
```

Comment:
Are these set on the `Organization` object? We don't set them and I don't know if Dapper updates the object.

In any case, I think it's more accurate to use `DateTime.Now` here.

Reply:
> Are these set on the `Organization` object?

They are, both properties are set when an instance is created:
```
public DateTime CreationDate { get; set; } = DateTime.UtcNow;
public DateTime RevisionDate { get; set; } = DateTime.UtcNow;
```

The same is true for `OrganizationUser` entities so I just removed these lines.